src/java/org/apache/cassandra/db/guardrails/EnableFlag.java

```diff
@@ -24,35 +24,35 @@
 import org.apache.cassandra.service.ClientState;
 
 /**
- * A guardrail that completely disables the use of a particular feature.
+ * A guardrail that enables the use of a particular feature.
  *
- * <p>Note that this guardrail only aborts operations (if the feature is disabled) so is only meant for
+ * <p>Note that this guardrail only enables operations (if the feature is enables) so is only meant for
```

Comment:
The original content `only aborts operations (if the feature is disabled)` is easier to understand.